File: JavaCalculator/src/main/java/com/softuni/entity/Calculator.java
package com.softuni.entity;

public class Calculator
{
    private double leftOperand;
    private double rightOperand;
    private String operator;

    public Calculator(double leftOperand, double rightOperand, String operator) {
        this.leftOperand = leftOperand;
        this.rightOperand = rightOperand;
        this.operator = operator;
    }


    public double getLeftOperand() {
        return leftOperand;
    }

    public void setLeftOperand(double leftOperand) {
        this.leftOperand = leftOperand;
    }

    public double getRightOperand() {
        return rightOperand;
    }

    public void setRightOperand(double rightOperand) {
        this.rightOperand = rightOperand;
    }

    public String getOperator() {
        return operator;
    }

    public void setOperator(String operator) {
        this.operator = operator;
    }

    public double calculateResult(){
        double result=0;

        switch (this.operator){
            case "+":
                result=this.leftOperand+this.rightOperand;
                break;
            case "-":
                result=this.leftOperand-this.rightOperand;
                break;
            case "*":
                result=this.leftOperand*this.rightOperand;
                break;
            case "/":
                result=this.leftOperand/this.rightOperand;
                break;

            case"!":
                result=1;
                for (int i=1; i<=this.leftOperand;i++) {
                    result *= i;
                }
                break;
            case"^":
                result=Math.pow(this.leftOperand,this.rightOperand);
                break;
            case"sqr(x)":
                result=Math.sqrt(this.leftOperand);
                break;

            case"%":
                result=this.rightOperand * (this.leftOperand/100.0);
                break;

                default:
                    result=0;
                    break;
        }

        return result;
    }
}

File: PHPCalculator/src/CalculatorBundle/Entity/Calculator.php
<?php
namespace CalculatorBundle\Entity;

class Calculator
{
    /**
     * @var float
     */
    private $leftOperand;
    /**
     * @var float
     */
    private $rightOperand;
    /**
     * @var string
     */
    private $operator;

    /**
     * @return float
     */
    public function getLeftOperand():?float
    {
        return $this->leftOperand;
    }

    /**
     * @param float $leftOperand
     */
    public function setLeftOperand($leftOperand)
    {
        $this->leftOperand = $leftOperand;
    }

    /**
     * @return float
     */
    public function getRightOperand(): ?float
    {
        return $this->rightOperand;
    }

    /**
     * @param float $rightOperand
     */
    public function setRightOperand($rightOperand)
    {
        $this->rightOperand = $rightOperand;
    }

    /**
     * @return string
     */
    public function getOperator():?string
    {
        return $this->operator;
    }

    /**
     * @param string $operator
     */
    public function setOperator($operator)
    {
        $this->operator = $operator;
    }

    /**
     * @return float
     */
    public function calculateResult():?float
    {
        $result=0;
        switch ($this->getOperator()){
            case"+":
            $result=$this->getLeftOperand()+$this->getRightOperand();
            break;
            case"-":
                $result=$this->getLeftOperand()-$this->getRightOperand();
                break;
            case"*":
                $result=$this->getLeftOperand()*$this->getRightOperand();
                break;
            case"/":
                $result=$this->getLeftOperand()/$this->getRightOperand();
                break;

            case"!":
                $result=1;
                for ($i=1;$i<=$this->getLeftOperand();$i++) {
                $result *= $i;
            }
            break;
            case"^":
                $result=pow($this->getLeftOperand(),$this->getRightOperand());
                break;
            case"sqr(x)":
                $result=sqrt($this->getLeftOperand());
                break;

            case"%":
                $result=$this->getRightOperand() * ($this->getLeftOperand()/100.0);
                break;
            default:
                break;

       }
        return $result;
    }


}